portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -5,6 +5,8 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
```

Comment:
You can probably change this to "/quote" as well as the name of this file, since this mainly handling quotes.

Reply:
Sorry, I meant to highlight line 10 with the WebServlet handle name,